src/extensions/bramble-extensions.json

```diff
@@ -8,6 +8,9 @@
         }
     },
     {
+        "path": "extensions/extra/HTMLHinter"
```

Comment:
A few things you need to do here:

1. Move this to `src/extensions/default` instead of `../extra`.  Extensions in `../extra` don't get statically cached.
2. Copy the images over:
```json
    "copy": [
        "extensions/default/HTMLHinter/images/*.*"
    ],
```
3. Build the CSS from your LESS:
```json
    "less": {
        "dist/extensions/default/HTMLHinter/main.css": [
            "dist/extensions/default/HTMLHinter/main.less"
        ]
    }
```

And then try and do `npm run build` and loading Brackets from `dist/` vs. `src/`, and make sure everything loads properly.